web/src/components/tasks/business-formation/review/ReviewStep.tsx

```diff
@@ -25,7 +25,9 @@ export const ReviewStep = (): ReactElement => {
   const { Config } = useConfig();
 
   const isLP = state.formationFormData.legalType === "limited-partnership";
-  const isForeignCCorp = state.formationFormData.legalType === "foreign-c-corporation";
+  const isForeignCorp = ["foreign-c-corporation", "foreign-s-corporation"].includes(
```

Comment:
Might be nice to make this a helper function and use it in `MainBusiness` also? idk

Reply:
and also in `validatedFieldsForUser`

Reply:
That's not a bad idea. I didn't for the sake of being explicit, but `isForeignCorp` should be enough in most places. I'll add that.